conan/cli/commands/audit.py

```diff
@@ -32,6 +43,8 @@ def audit_scan(conan_api: ConanAPI, parser, subparser, *args):
     # TODO: Do we then want to hide it in the --help?
     subparser.add_argument("--build-require", action='store_true', default=False,
                            help='Whether the provided reference is a build-require')
+    subparser.add_argument("-sl", "--severity-level", action="store", default=9.0, type=float,
```

Comment:
I'd add this for the `conan audit list` command too

Reply:
Maybe not necessary? What is the value or automation in that case? Are you going to scan nightly for ``conan audit list openssl/1.1.1`` and want to get an error in case it has a new vulnerability? I see the case for ``conan audit scan`` but not so much for ``conan audit list`` (not against adding it, but seems not as relevant)